docker/container-test.sh

```diff
@@ -33,3 +33,10 @@ function test_container {
     # Return proper exit code.
     [[ ${RESULT} == '"success"' ]]
 }
+
+function test_javahome {
+    # Check if JAVA_HOME works
+    docker run --rm --entrypoint /bin/bash "${CONTAINER_NAME}" -c "\$JAVA_HOME/bin/java -version" &> /dev/null
```

Comment:
Do we need to override the entry point? It seems like running the command directly works:
```
docker run --rm "${CONTAINER_NAME}" /bin/bash -c '$JAVA_HOME/bin/java -version' &> /dev/null
```
If we use `'` instead of `"` we don't need to escape the `$` in the argument.

Reply:
no need to override the entrypoint since I saw it uses `CMD` instruction only in `Dockerfile`.